shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DDLStatement.g4

```diff
@@ -305,3 +474,514 @@ indexExistClause_
 indexNames
     : indexName (COMMA_ indexName)*
     ;
+
+alterDatabase
+    : ALTER DATABASE databaseName alterDatabaseClause
+    ;
+
+alterDatabaseClause
+    : (WITH)? createdbOptItems?
+    | RENAME TO databaseName
+    | OWNER TO roleSpec
+    | SET TABLESPACE name
+    | setResetClause
+    ;
```

Comment:
Plz refer to gram.y, maybe the official doc is not correct.
![image](https://user-images.githubusercontent.com/16559542/89881268-da0e5980-dbf7-11ea-8812-3ff7f722f29e.png)